test/python/expected/sql_validation_framework/expected_create.out

```diff
@@ -47,6 +47,7 @@ Could not find tests for function sys.suser_id_internal
 Could not find tests for function sys.suser_name_internal
 Could not find tests for function sys.sysdatetime
 Could not find tests for function sys.timefromparts
+Could not find tests for function sys.tsql_get_expr
```

Comment:
entry in this file indicates that tests are missing for these cases, please cover the same

Reply:
Added testcase regarding sys.tsql_get_expr.